lib/turn.rb

```diff
@@ -7,18 +7,22 @@ def compare(guess)
     colors_correct = 0
     colors_in_correct_positions = 0
 
-    guess = 'rrgb'
-    array = guess.split ''
-    p array
+    hidden_code = ['g', 'r', 'r', 'b']
+    guess = 'rrgb' # This will probably wind up being equal to guess.gets from the game class
```

Comment:
Assuming you meant to comment this whole line out cause right now it's reassigning the guess argument before we check it